src/Symfony/Component/Notifier/Bridge/Slack/SlackTransportFactory.php

```diff
@@ -28,17 +29,20 @@ final class SlackTransportFactory extends AbstractTransportFactory
      */
     public function create(Dsn $dsn): TransportInterface
     {
-        $scheme = $dsn->getScheme();
+        if ('slack' !== $dsn->getScheme()) {
+            throw new UnsupportedSchemeException($dsn, 'slack', $this->getSupportedSchemes());
+        }
+
+        if ('/' !== $dsn->getPath()) {
+            throw new InvalidArgumentException(sprintf('Invalid "%s" notifier DSN: Using a dsn for slack webhooks has been removed (maybe you haven\'t update the DSN when upgrading from 5.1).', $dsn->getOriginalDsn()));
```

Comment:
Maybe better to use `throw new IncompleteDsnException('Support for slack webhook dsn has been dropped since 5.2 (maybe you haven\'t updated the DSN when upgrading from 5.1).');`?

Reply:
I agree.

Please, change "webhook dsn" to "webhook DSN".

Reply:
But Incomplete... is missleading, it's a wrong DSN... 🧐

Reply:
@OskarStark That was also my first thought but i wasn't sure if it is okay if i a add a new exception.